src/dyad/dtl/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# DTL Interface
+set(DTL_SRC ${CMAKE_CURRENT_SOURCE_DIR}/dyad_dtl_impl.c)
+set(DTL_PRIVATE_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/dyad_dtl_impl.h
+                        ${CMAKE_CURRENT_SOURCE_DIR}/dyad_rc.h)
+set(DTL_PUBLIC_HEADERS )
```

Comment:
See the comment for `CMakeLists.txt` in `core`

Reply:
We dont need it as core is not in public header